src/app/screens/week-summary/index.tsx

```diff
@@ -1,16 +1,19 @@
 import React, { useEffect, useState } from 'react';
 import { useTranslation } from 'react-i18next';
 import { ScrollView } from 'react-native';
+import { NavigationProp, useNavigation } from '@react-navigation/native';
 
 import { Button, Heading, ScreenContainer, Text } from 'src/app/components';
+import { SummaryCard } from 'src/app/components/summary-card';
+import { MainDrawerParamList } from 'src/app/navigation/types';
 import { WeekSummaryData } from 'src/interfaces/week-summary';
 import weekSummaryDataFile from 'src/mocks/week-summary-data.json';
 
-import { SummaryCard } from './components/summary-card';
 import { CardsContainer } from './styles';
 
 export const WeekSummaryScreen = () => {
   const { t } = useTranslation();
+  const navigation = useNavigation<NavigationProp<MainDrawerParamList>>();
```

Comment:
If you are inside a screen, you should use the navigation that comes from propsa